cms/djangoapps/contentstore/views/videos.py

```diff
@@ -428,7 +428,7 @@ def make_csv_dict(video):
     writer = csv.DictWriter(
         response,
         [
-            col_name.encode("utf-8")
+            col_name.encode("utf-8") if six.PY2 else col_name
```

Comment:
Why don't we encode the column names in python3, won't the DictWriter still expect bytes here?